__tests__/app.tests.js

```diff
@@ -98,31 +98,47 @@ describe("GET api/article", () => {
       .get("/api/articles")
       .expect(200)
       .then((res) => {
-        const output = res.body.articles[0];
-        expect(typeof res.body).toEqual("object");
-        expect(output).toHaveProperty("author");
-        expect(output).toHaveProperty("title");
-        expect(output).toHaveProperty("article_id");
-        expect(output).toHaveProperty("topic");
-        expect(output).toHaveProperty("created_at");
-        expect(output).toHaveProperty("votes");
-        expect(output).toHaveProperty("article_img_url");
+        const articles = res.body.articles;
+        articles.forEach((article) => {
```

Comment:
Can we check the length of the articles array so we know the callback used by forEach is executed. Same comment for where it is used elsewhere.

This test seems unnecessary also as we assert against each key in the following test.

Do you check the elements are sorted at all? Have a look at jest-sorted if you haven't yet